src/common/constants.js

```diff
@@ -1,6 +1,6 @@
 import * as yup from "yup";
 
-export const schema = yup.object().shape({
+export const userSchema = yup.object().shape({
```

Comment:
We will not use the same schema for the login and signup forms. Because signup form should have a confirm password field that checks whether the passwords match.

What you can do is this:
1- Create a constants folder in the src folder.
2- add validations.js file to constants folder
3- add the validations here like the following:

```js
export const stuff = yup
    .string()
    .required("Lütfen işletmenizin ismini girin.")
    .min(3, "İşletme ismi 3 veya daha fazla karakterden oluşmalı.")
```

So you can use the same validation in schemas.